p2p/module.go

```diff
@@ -0,0 +1,53 @@
+package pre_p2p
+
+import (
+	"log"
+	"pocket/shared/config"
+	"pocket/shared/modules"
+
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+var _ modules.NetworkModule = &networkModule{}
+
+type networkModule struct {
+	pocketBus modules.Bus
+	p2pConfig *config.P2PConfig
+}
+
+func Create(config *config.Config) (modules.NetworkModule, error) {
+	return &networkModule{
+		pocketBus: nil,
+		p2pConfig: config.P2P,
+	}, nil
+}
+
+func (p *networkModule) Start() error {
+	// TODO(olshansky): Add a test that pocketBus is set
+	log.Println("Starting PRE P2P module...")
+	return nil
+}
+
+func (p *networkModule) Stop() error {
+	log.Println("Stopping PRE P2P module...")
```

Comment:
I see p2p and I see network

We need terminology to be completely unified before moving forward

Reply:
Done. Using `p2p` everywhere now.